public/media-hunt/commands/movies/toprated.js

```diff
@@ -1,8 +1,19 @@
 const { getMovieTopRated } = require("../../http/movie");
+const { handleDatabase } = require("../../database/database");
 
 const run = async () => {
-    const data = await getMovieTopRated();
-    console.log(data);
-  };
+  {
+    let data;
+    if (handleDatabase.checkIfExists()) {
+      data = handleDatabase.getDataByKey();
+    } else {
+      const response = await getMovieTopRated();
+      data = handleDatabase.storeDataByKey(response);
+    }
+    console.log("data here", data);
+  }
+  const data = await getMovieTopRated();
+  console.log(data);

```

Comment:
In every one of these, you have a lot of errors:
```
  {
    let data;
    if (handleDatabase.checkIfExists()) {
      data = handleDatabase.getDataByKey();
    } else {
      const response = await getMovieTopRated();
      data = handleDatabase.storeDataByKey(response);
    }
    console.log("data here", data);
  }
  const data = await getMovieTopRated();
  console.log(data);
```

1. You don't need first `{}` pair
2. You have always 2 console logs, always HTTP request, since you didn't get away from performing HTTP calls to get data. You just log it twice. :/
3. The whole entire `{}` content from line 6 to line 13 basically needs to replace the bottom two lines. In every command.